api/client/start.go

```diff
@@ -32,6 +32,7 @@ func (cli *DockerCli) forwardAllSignals(cid string) chan os.Signal {
 			}
 			if sig == "" {
 				fmt.Fprintf(cli.err, "Unsupported signal: %v. Discarding.\n", s)
+				continue
```

Comment:
@runcom I think that default has been there since we merged the STOPSIGNAL in, which was a few weeks ago.

@Mashimiao I think you can replace all this loopy logic to find the signal with a call to `signal.ParseSignal`. It was added recently too:

https://github.com/docker/docker/blob/fix-bug-client-start-signal-discarded/pkg/signal/signal.go#L31

But the `continue` looks good, anyways.